packages/2d/src/lib/components/Latex.ts

```diff
@@ -36,6 +41,14 @@ const JaxDocument = mathjax.document('', {
   OutputJax: new SVG({fontCache: 'local'}),
 });
 
+// presets required for the Pitex animations to work
+const FACTOR = 2;
+const BUFFER = 0.25;
```

Comment:
I prefer to inline these into the functions, especially when they're just used to cut a duration in half or in quarters. That makes it easier to change an individual function without ramifications over all of the other durations in the file.